eventdiscovery-sdk/src/main/res/values/dimens.xml

```diff
@@ -13,4 +13,5 @@
     <dimen name="schedjoules_list_item_default_height">96dp</dimen>
     <dimen name="schedjoules_event_list_progress_bar">36dp</dimen>
     <dimen name="schedjoules_event_detail_item_icon">28dp</dimen>
+    <dimen name="schedjoules_toolbar_title">20sp</dimen>
```

Comment:
You are right, that wasn't a good enough name, changed it and also changed `sp` to `dp`, I looked it up incorrectly earlier it seems, because I just wanted to match it with the theme (it was needed for a bug about smaller text size when screen rotated). But I clicked through now and the android theme has:
```
    <dimen name="text_size_title_material_toolbar">20dp</dimen>
```

Regarding the naming format, I agree and like the way to combine underscore and camel-case, but I would suggest we create a separate story for it to update every resource. First, define it for all resource types and then implement when there is not much parallel work in progress task, to avoid the conflicts.
I actually would like the first letters capital as well, it looked strange first when I saw that, but after I got used to it looked better actually, so something like this:
`SchedJoules_EventList_ProgressBar`
but this context first may not be best for all resource type I think, for example the order above for the text size in android theme might make sense also as well. So I think we should just discuss and defined the formats in a backlog story first. Okay to create it?

(To not mix formats, I renamed this dimen to `schedjoules_toolbar_title_font_size` now.)